integration-tests/src/test/java/org/apache/fineract/integrationtests/common/GlobalConfigurationHelper.java

```diff
@@ -99,8 +99,8 @@ public static void verifyAllDefaultGlobalConfigurations(final RequestSpecificati
         ArrayList<HashMap> actualGlobalConfigurations = getAllGlobalConfigurations(requestSpec, responseSpec);
 
         // There are currently 37 global configurations.
-        Assertions.assertEquals(38, expectedGlobalConfigurations.size());
-        Assertions.assertEquals(38, actualGlobalConfigurations.size());
+        Assertions.assertEquals(39, expectedGlobalConfigurations.size());
+        Assertions.assertEquals(39, actualGlobalConfigurations.size());
```

Comment:
this also requires updating the Map for teardown. in `getAllDefaultGlobalConfigurations` method of `GlobalConfigurationHelper`